systemd/systemd.go

```diff
@@ -437,12 +437,12 @@ func MountUnitPath(baseDir string) string {
 }
 
 func (s *systemd) WriteMountUnitFile(name, what, where, fstype string) (string, error) {
-	extra := ""
+	options := []string{"nodev"}
```

Comment:
@pedronis devices in cgroup is totally unrelated, it only works when tagging is used. On core I don't know how this happens but we mount core from the initrd. On classic we overmount /dev so maybe this PR can go in as-is.